### DISCUSSION TITLE
反代端口建立网站，设置网站密码访问后，输入密码登陆总是失败

### DISCUSSION THREAD

**Discussion by maskgo68** [1Panel-dev/1Panel#5068] [Q&A]
### 1Panel 版本

社区版：V1.10.5-lts

### 问题描述

反代端口建立网站，设置网站密码访问后，在浏览器打开网站，输入设置好的用户名密码，登陆总是失败，马上继续弹出密码登陆窗口，一直要求重新输入
![image](https://github.com/1Panel-dev/1Panel/assets/146368179/ec354daa-a656-46d3-8a97-b4f8aea403a8)

已测试：edge、chrome，清除浏览器缓存，都存在问题

### 重现步骤

反代端口建立网站，设置网站密码访问后，在浏览器打开网站，输入设置好的用户名密码，登陆总是失败，马上继续弹出密码登陆窗口，一直要求重新输入

已测试：edge、chrome，清除浏览器缓存，都存在问题

**Comment by zhengkunwang223** [comment]
后端是什么呢？
考虑跟后端的校验冲突了

**Reply by adrve** [reply]
> 后端是什么呢？ 考虑跟后端的校验冲突了

社区版：V1.10.11-lts，我的后端是ESXI，也遇到了这种情况
